app/controllers/submissions_controller.rb

```diff
@@ -35,11 +35,7 @@ def new
         render([@course, @assessment, :submissions]) && return
       end
     else
-      @cuds = {}
-      # TODO: change order
-      @course.course_user_data.joins(:user).order("email ASC").each do |cud|
-        @cuds[cud.full_name_with_email] = cud.id
-      end
+      retrieve_autocompletion_data!
```

Comment:
The `.joins(:user).order("email ASC")` appears redundant, so this should be fine

Reply:
Mainly because order doesn't really matter when we're using the autocomplete widget